.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+Pull Request Checklist
+ - [ ] Pull request includes a description of the change and the reason behind it.
+ - [ ] Pull request [uses keywords to close relevant issues](https://help.github.com/en/articles/closing-issues-using-keywords).
+ - [ ] Pull request includes unit tests for any new functionality.
```

Comment:
You can enforce this with `.codecov` (a free integration for coverage testing). For example, see [how I did this for a side project I'm working on](https://github.com/jameslamb/doppel-cli/blob/master/.codecov.yml#L7).

This part at that link means "fail builds if less than 20% of new code added is uncovered by tests"

```{yaml}
coverage:
  status:
    patch:
      default:
        target: '20'
```

I still like having the line here in the pull request template, but just know that you might be able to remove it once someone picks up #71 . I'm a big fan of rejecting PRs because of failed builds (rejection by automation) wherever possible, instead of relying on checklists (rejection by convention).

Reply:
I'm with you for packages. I'm not sure about this repo, which currently consists of pretty simple scripts and notebooks that generally don't lend themselves well to automated tests.